src/components/Transaction.vue

```diff
@@ -42,16 +106,17 @@ export default {
 <style lang="scss" scoped>
 @import '@/styles/_colors.scss';
 @import '@/styles/theme.scss';
+.border {
```

Comment:
I think we should also add the `border-bottom` to transactions.

![image](https://user-images.githubusercontent.com/24905681/80363846-1d310080-8885-11ea-9f96-cd3c4920ecda.png)